test/session_test.dart

```diff
@@ -376,7 +378,8 @@ Future<void> main([List<String>? args]) async {
           final sessionError = syncError.as<SyncWebSocketError>();
           expect(sessionError.category, SyncErrorCategory.webSocket);
           expect(sessionError.code, errorCode);
-          expect(sessionError.message, "Simulated session error");
+          expect(sessionError.detailedMessage, "Simulated session error");
+          expect(sessionError.detailedMessage == sessionError.message, isFalse);
```

Comment:
```suggestion
      expect(sessionError.detailedMessage, isNot(equals(sessionError.message)));
```
? 
then we get nicer and more descriptive error message if this fails the test

Reply:
also should work
```dart
expect(sessionError.detailedMessage, isNot(sessionError.message));
```